### DISCUSSION TITLE
Add Philips Hue Bulbs to Zigbee Network

### DISCUSSION THREAD

**Discussion by Shaun-Harrison** [Koenkk/zigbee2mqtt#10271] [General]
I've recently bought a Sonoff Zigbee USB Bridge - get a couple of devices set up via Zigbee2MQTT to work in Home Assistant 

I have philips hue bulbs around the house, all of which are setup to the philips hue bridge

Not sure if this is a Zigbee2MQTT specific question, but is it possible to add the bulbs to my Sonoff Bridge while still having them linked to the hue bridge ?

Reason for this is to keep existing philips hue functionality such as turning lights on with google home and so on, but would be good to get them added to my own zigbee network, so that the mesh is improved

**Comment by ajfriesen** [comment]
I just moved my hue bulbs to my ZigBee2MQTT sonoff Stick.

I think it is technically not possible to have them take to two ZigBee networks at the same time.
